plugins/obs-outputs/data/locale/en-US.ini

```diff
@@ -15,7 +15,8 @@ HostNotFound="Hostname not found. Make sure you entered a valid streaming server
 NoData="Hostname found, but no data of the requested type. This can occur if you have bound to an IPv6 address and your streaming service only has IPv4 addresses (see Settings → Advanced)."
 AddressNotAvailable="Address not available. You may have tried to bind to an invalid IP address (see Settings → Advanced)."
 SSLCertVerifyFailed="The RTMP server sent an invalid SSL certificate."
-InvalidParameter="Invalid connection parameters. Check your server address and port as well as IP address settings (see Settings → Advanced)."
+InvalidParameter="Invalid connection parameters. Check that the streaming service address is correct."
+NoRoute="Error reaching host. Make sure that the interface you have bound can access the internet and that the streaming service supports the address family you selected (see Settings → Advanced)."
```

Comment:
For reviewers: The NoRoute error will _only_ be thrown when the user has bound an IP address or has explicitly chosen IPv4-Only or IPv6-Only. This will not happen if they have left or chosen the defaults.